rktl_sim/src/simulator/sim.py

```diff
@@ -181,25 +181,19 @@ def step(self, throttle_cmd, steering_cmd, dt):
                     self.scored = True
                     self.winner = "B"
 
-        # Step kinematic objects independently
-        for car in self._cars.values():
-            wallContact = False
-            carContacts = p.getContactPoints(bodyA=car.id)
-            for contact in carContacts:
-                if contact[2] in self._walls:
-                    wallContact = True
-                    break
-            car.step((throttle_cmd, steering_cmd), wallContact, dt)
-    
         # PyBullet steps at 240hz
-        for _ in range(int(dt * 240.)):
+        p_dt = 1.0 / 240.0
+        for _ in range(round(dt / p_dt)):
+            # Step kinematic objects independently, at max possible rate
```

Comment:
Everything is ready to be merged. Only possible hiccup is this. The bicycle model stuff does greatly benefit from being run at a rate higher than 10 Hz, but 240 might be excessive.

Reply:
See `setTimeStep` in the [quickstart guide](https://usermanual.wiki/Document/PyBullet20Quickstart20Guide.543993445/html)

Reply:
If PyBullet is best tuned for 240Hz, I probably don't want to change that. Instead we could look at instead maybe stepping these at 30 or 60 Hz inside that loop. But I figure we only do that if we realize it is slowing stuff down.